### PR TITLE
Comments on the draft of the 25/01/2023

### PR DESCRIPTION
**Pull request by bkhelifi**
Hi all,
I put the annotated PDF, as well as the modifications directly into the files.

### CODE REVIEW

**Comment by adonath** [comment]
Thanks @bkhelifi, I think you accidentally updated the Gammapy submodule which should remain at v1.0. Please fix...otherwise looks good to me.

File: src/ms-review.tex [hunk 1]
```diff
@@ -456,21 +460,23 @@ \section{Gamma-ray Data Analysis}
 
 \gammaray data at the Data Level 3 (DL3) therefore consist of lists of gamma-like events and their
 corresponding instrument response functions (IRFs). The latter include the effective area ($A_{\rm eff}$),
-point spread function (PSF) and energy dispersion ($E_{\rm disp}$).
+point spread function (PSF) and energy dispersion ($E_{\rm disp}$). In general, they depend on event's detector
+geometrical parameters, e.g. the field-of-view location or the event elevation angle. So they might be parametrised as
+function of such parameters specific to the instrumental technical.
 
-An additional component of DL3 IRFs is the residual hadronic background model $Bkg$.
+An additional component of DL3 IRFs is the model if residual hadronic background rate, $Bkg$.
```
Comment (AtreyeeS) [inline src/ms-review.tex:467]:
```suggestion
An additional component of DL3 IRFs is a model of the residual hadronic background rate, $Bkg$.
```

File: src/ms-review.tex [hunk 2]
```diff
@@ -1879,24 +1887,18 @@ \section{Summary and Outlook}
...
 \begin{acknowledgements}
 
-	Mention Christoph here?
-
 	We would like to thank the \texttt{Numpy}, \texttt{Scipy}, \texttt{IPython} and
 	\texttt{Matplotlib} communities for providing their packages which are
-	invaluable to the development of Astropy. We thank the \github team for
+	invaluable to the development of Gammapy. We thank the \github team for
 	providing us with an excellent free development platform. We also are grateful
 	to Read the Docs (\ReadthedocsUrl), and Travis (\TravisUrl) for providing free
```
Comment (AtreyeeS) [inline src/ms-review.tex:1894]: do we still use read the docs?

**Review by AtreyeeS — APPROVED** [review]
Thanks @bkhelifi  for this very detailed review

File: src/ms-review.tex (hunk 2 above)
Comment (adonath) [reply src/ms-review.tex:1894]: No, but we used it for a long time, as well as Travis so probably fine to keep...

**Comment by bkhelifi** [comment]
PR closed and transfered into the PR #85.

**Pull request closed without merging**